runtime/kusama/src/weights/pallet_balances.rs

```diff
@@ -93,4 +93,7 @@ impl<T: frame_system::Config> pallet_balances::WeightInfo for WeightInfo<T> {
 			.saturating_add(T::DbWeight::get().reads(1 as u64))
 			.saturating_add(T::DbWeight::get().writes(1 as u64))
 	}
+	fn upgrade_accounts(_: u32) -> Weight {
+		Weight::from_parts(0, 0)
```

Comment:
Weights will be updated before the next release, but I will still copy them over from Substrate - just in case.